iceoryx_platform/.clang-tidy

```diff
@@ -0,0 +1 @@
+../iceoryx_hoofs/include/iceoryx_hoofs/cxx/.clang-tidy
```

Comment:
I'm not sure this is a good idea. We already had issues with paths above the current working directory

Reply:
You are right this is a bad solution. But there was already a clang-tidy symlink in platform which relatively pointed to exactly this file and I just corrected the path.

I would suggest that this stays as it is since this approach can also be found in other directories and we create an issue to clean this up. This is not part of this PR.